1.18/root/opt/app-root/nginxconf.sed

```diff
@@ -1,4 +1,4 @@
-/listen/s%80%8080%
+/listen/s%80%8080 default_server%
```

Comment:
Only one question. This is relevant only for version `1.18`? What about the rest?

Reply:
It is relevant only for 1.18. Then LGTM.